.github/workflows/qc.yml

```diff
@@ -12,7 +12,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, windows-latest]
-        python-version: [ "3.9", "3.10", "3.11", "3.12" ]
+        python-version: [ "3.10", "3.11", "3.12" ]
```

Comment:
Shouldn’t we use that opportunity to add `3.13`?

Reply:
Strange that the test should fail under 3.13, though. The test suite runs fine under 3.13 on my machine.

Reply:
I don't want to start working on fixing (removing) poetry in this PR.

Reply:
> fixing (removing) poetry

🤣 Fair enough!